public/chromeos.html

```diff
@@ -0,0 +1,282 @@
+                />
+              </svg>
+              <p>Switch</p>
+            </a>
+          </li>
+          <li>
+            <a href="/xbox">
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 372.4 372.6">
+                <path
+                  class="svg-fill"
+                  d="M169.2,359.4c-28.7-2.7-57.7-13-82.7-29.3
+                c-20.9-13.6-25.6-19.2-25.6-30.4c0-22.5,24.7-61.8,67-106.7c24-25.5,57.5-55.4,61.1-54.6c7,1.6,63.3,56.4,84.3,82.2 c33.3,40.8,48.6,74.2,40.8,89.1c-5.9,11.3-42.6,33.5-69.5,42C222.4,358.8,193.2,361.7,169.2,359.4L169.2,359.4z M32.7,276.3 c-17.4-26.6-26.1-52.9-30.4-90.8c-1.4-12.5-0.9-19.7,3.2-45.4c5.1-32.1,23.3-69.1,45.3-92c9.3-9.7,10.2-10,21.6-6.1 c13.8,4.7,28.6,14.9,51.5,35.6l13.4,12.1l-7.3,9C96,140.2,60.2,199.2,46.7,235.7c-7.3,19.8-10.3,39.8-7.1,48 C41.7,289.3,39.8,287.3,32.7,276.3z M337.6,280.9c1.7-8.4-0.5-23.8-5.5-39.3c-11-33.6-47.8-96.1-81.7-138.6l-10.6-13.4L251.3,79 c15-13.8,25.5-22.1,36.7-29.1c8.9-5.5,21.6-10.4,27.1-10.4c3.4,0,15.2,12.3,24.8,25.7c14.8,20.8,25.7,46,31.2,72.2 c3.6,16.9,3.9,53.2,0.6,70.1c-2.7,13.9-8.4,31.9-14,44.1c-4.2,9.1-14.5,26.9-19.1,32.7C336.3,287.3,336.3,287.3,337.6,280.9 L337.6,280.9z M172.3,33.1c-15.6-7.9-39.7-16.4-53-18.7c-4.7-0.8-12.6-1.2-17.7-1c-11,0.6-10.5,0,7.1-8.3 c14.6-6.9,26.8-11,43.4-14.5c18.6-3.9,53.7-4,72-0.1c19.8,4.2,43.1,12.9,56.3,20.9l3.9,2.4l-9-0.5c-17.8-0.9-43.8,6.3-71.6,19.9
+                c-8.4,4.1-15.7,7.4-16.3,7.3C187,40.4,180.1,37.1,172.3,33.1z"
+                />
+              </svg>
+              <p>Xbox</p>
+            </a>
+          </li>
+          <li>
+            <a href="/playstation">
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 154.8">
+                <path
+                  class="svg-fill"
+                  d="M197.2,118c-3.9,4.9-13.3,8.4-13.3,8.4l-70.5,25.3V133l51.9-18.5c5.9-2.1,6.8-5.1,2-6.7 c-4.8-1.6-13.4-1.1-19.3,1L113.4,121v-19.4l2-0.7c0,0,10-3.5,24-5.1c14-1.5,31.2,0.2,44.7,5.3C199.4,106,201.1,113.1,197.2,118z M120.1,86.2V38.4c0-5.6-1-10.8-6.3-12.2c-4-1.3-6.5,2.5-6.5,8.1v119.6L75,143.6V1c13.7,2.5,33.7,8.6,44.4,12.2 c27.3,9.4,36.6,21,36.6,47.3C156,86.2,140.2,95.9,120.1,86.2z M15.9,131c-15.6-4.4-18.2-13.6-11.1-18.8c6.6-4.9,17.8-8.5,17.8-8.5 l46.2-16.4v18.7l-33.3,11.9c-5.9,2.1-6.8,5.1-2,6.7c4.8,1.6,13.4,1.1,19.3-1l16-5.8v16.8c-1,0.2-2.1,0.4-3.2,0.5 C49.6,137.6,32.6,136.5,15.9,131L15.9,131z"
+                />
+              </svg>
+              <p>PS4/5</p>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1 class="main-title">How to take a screenshot on ChromeOS</h1>
+      <article id="full">
+        <section class="intro">
+          <h1 class="intro-title">Screenshot Region</h1>
+          <p>Some chromebooks have a dedicated <kbd>Screenshot</kbd> key.</p>
```

Comment:
Can we make sure that Chromebook is capitalised? This applies in other places where the name appears.

```suggestion
          <p>Some Chromebooks have a dedicated <kbd>Screenshot</kbd> key.</p>
```